llvm/build.mk

```diff
@@ -20,6 +23,8 @@ $(LLVM_BUILD)/$(if $(NINJA),build.ninja,Makefile): $(abs_top_srcdir)/external/ll
 	cd $(LLVM_BUILD) && $(CMAKE) \
 		$(if $(NINJA),-G Ninja) \
 		-DCMAKE_INSTALL_PREFIX="$(LLVM_PREFIX)" \
+		-DCMAKE_C_FLAGS="$(CMAKE_C_FLAGS)" \
```

Comment:
In a different project that has similar requirements we also added this to the cmake flags, don't know if it is needed when you add the C/CCX FLAGS, just validate that the final build opt.exe, llc.exe etc doesn't have any additional dependecy on c/c++ runtimes.

`-DCMAKE_EXE_LINKER_FLAGS="-static"`

Reply:
In that case we only used that flag together with the -all-static when linking mono runtime and that produced a cross compiler without any additional dependecies.